web/modules/custom/dpl_redia_legacy/src/RediaEvent.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Drupal\dpl_redia_legacy;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\dpl_event\EventWrapper;
+use Drupal\node\NodeInterface;
+use Drupal\recurring_events\Entity\EventInstance;
+
+/**
+ * An event object, containing the properties the RSS feed needs.
+ */
+class RediaEvent extends ControllerBase {
+  // We'll disable the documentation rules for the member properties, as
+  // they are pretty self-explanatory.
+  // phpcs:disable
```

Comment:
Consider if is it possible to only [disable the related rules for these lines](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file) instead of PHPCS entirely.

Reply:
I did try, and it just refused to accept it, so this is me going nuclear.